shared/test/integration/helpers/view.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+var Marionette = require('marionette-client');
+
+function View(client) {
+    this.client = client.scope({searchTimeout: 5000});
+    this.actions = new Marionette.Actions(this.client);
+}
+
+View.prototype = {
+    getRootElement: function (rootSelector, method) {
+        return this.client.findElement(rootSelector, method);
+    },
+    findElement: function (selectorObject) {
+        return this.client
+        .findElement(selectorObject.locator, selectorObject.by);
+            
+    },
+    rootDisplayed: function () {
+        return this.getRootElement().displayed();
+    },
+    waitForDisplay: function (selectorObject) {
+    	var selector = selectorObject;
+        var self = this;
+        return this.client.waitFor(function () {
+            return self.findElement(selector).displayed();
+        });
+    },
+    waitForHide: function () {
+        var self = this;
+        return this.client.waitFor(function () {
+            return !self.displayed();
+        });
+    },
+    tap: function (selectorObject) {
+        this.actions.tap(this.findElement(selectorObject)).perform();
```

Comment:
Why do we have tap in multiple places. I saw other tests using actions tap method. We need to pick one and just use that.